run_python_examples.sh

```diff
@@ -194,17 +195,15 @@ if [ "" == "$EXAMPLES" ]; then
 else
   for i in $(echo $EXAMPLES | sed "s/,/ /g")
   do
+    echo "Starting $i"
     $i
+    echo "Finished $i, status $?"
   done
 fi
 
 if [ "" == "$ERRORS" ]; then
-  tput setaf 2
-  echo "Completed successfully"
+  echo "Completed successfully with status $?"
 else
-  tput setaf 1
   echo "Some examples failed:"
   printf "$ERRORS"
 fi
-
-tput sgr0
```

Comment:
tput was erroring. The intent was to change the font color, but it does not work on the CI.